internal/cli/cmd/describe/describe.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package describe
+
+import (
+	"fmt"
+
+	cmdUtils "github.com/perses/perses/internal/cli/utils"
+	cmdUtilsService "github.com/perses/perses/internal/cli/utils/service"
+	modelV1 "github.com/perses/perses/pkg/model/api/v1"
+	"github.com/spf13/cobra"
+)
+
+type option struct {
+	kind            modelV1.Kind
+	project         string
+	name            string
+	output          string
+	resourceService cmdUtilsService.Service
+}
+
+func (o *option) complete(args []string) error {
+	if len(args) < 1 {
+		return fmt.Errorf(cmdUtils.FormatAvailableResourcesMessage())
+	} else if len(args) < 2 {
+		return fmt.Errorf("you have to specify the name of the resource you have to describe")
```

Comment:
```suggestion
		return fmt.Errorf("please specify the name of the resource you want to describe")
```

Sounds friendlier? :)

Reply:
Indeed, less threatening haha.